src/core/core.animations.js

```diff
@@ -1,121 +1,166 @@
 'use strict';
 
-var defaults = require('./core.defaults');
-var helpers = require('../helpers/index');
+const Animator = require('./core.animator');
+const Animation = require('./core.animation');
+const helpers = require('../helpers/index');
+const defaults = require('./core.defaults');
 
 defaults._set('global', {
 	animation: {
 		duration: 1000,
 		easing: 'easeOutQuart',
+		active: {
+			duration: 400
+		},
+		resize: {
+			duration: 0
+		},
+		numbers: {
+			type: 'number',
```

Comment:
it seems a little redundant to have to specify `type` when it's already specified as the key. Could we just use the key?

Reply:
The key name is arbitrary, I just ended up choosing `numbers`. It could be anything.

I'm actually not sure if this is a good way of specifying defaults. Maybe those should be under `defaults`, so those can be disabled easily:
```js
animation: {
  defaults: false,
  location: { // multiple properties targeted
    type: 'number',
    properties: ['x', 'y', 'base']
    easing: 'linear',
    duration: 200
  },
  dimensions: { // multiple
    type: 'number',
    properties: ['width', 'height'],
    duration: 400,
    easing: 'easeOutElastic'
  },
  multiKeyBackground: { // single
    type: 'color',
    easing: 'linear'
  }
}
```

But these things that can be altered when people start trying out things (alpha).